playbook.yml

```diff
@@ -178,10 +210,47 @@
       notify:
         - restart apache
 
+    # Nginx Symlink for 14.04 and higher.
+    - name: Symbolic link to Aegir's Nginx Config
+      when: server_webserver == "nginx" and (ansible_distribution != "Ubuntu" or ansible_lsb.major_release|int >= 14)
+      file:
+        src=/var/aegir/config/nginx.conf
+        dest={{ nginx_confd_path }}
+        state=link
+        force=yes
+      notify:
+        - restart nginx
+
+
+    - name: Symbolic link to Aegir's Nginx Config (When ubuntu < 14.04)
+      when: server_webserver == "nginx" and (ansible_os_family == "Debian" and ansible_lsb.major_release|int < 14)
+      file:
+        src=/var/aegir/config/nginx.conf
+        dest=/etc/nginx/sites-enabled/aegir.conf
+        state=link
+        force=yes
+      notify:
+        - restart nginx
+
+    - name: Write www.conf file. # @TODO: Restart apache if changed.
+      when: server_webserver == "nginx"
+      template:
+        src=templates/www.conf.j2
+        dest={{ nginx_www_pool_path }}
+        mode=0644
+
+    - name: Write php.ini file. # @TODO: Restart apache if changed.
+      when: server_webserver == "apache"
+      template:
+        src=templates/php.ini.j2
+        dest={{ apache_php_ini_path }}
+        mode=0644
+
     - name: Write php.ini file. # @TODO: Restart apache if changed.
+      when: server_webserver == "nginx"
       template:
         src=templates/php.ini.j2
-        dest={{ php_ini_path }}
+        dest={{ nginx_php_ini_path }}
         mode=0644
```

Comment:
Similar to the "devshop_packages" variable, I think it would be better to include a `nginx-variables.yml` file that overrides things like php_ini_path if using nginx.

Let's keep the duplicate items in playbook.yml to a minimum and just use variables whenever possible.